spec.bs

```diff
@@ -2238,3 +2240,85 @@ achieve the outcomes described in the above explanatory content.
   /fenced-frame/permission-geolocation.https.html
   /fenced-frame/permission-notification.https.html
 </wpt>
+
+<h3 id=private-network-access-monkeypatch>Private network access</h3>
+
+Private network access is a security feature that allows you to control which apps on your device can
+access your local network. On creation, regular iframes inherit their policy container from the
+navigation initiator, while fenced frames explicitly do not. While this prevents information from
```

Comment:
Do they _always_ get it from the initiator? What part of the spec are you referencing. For example, https://source.chromium.org/chromium/chromium/src/+/main:content/browser/renderer_host/navigation_request.cc;l=4933-4941;drc=41b3f66441ca114b9b75194a635b9db7b588b772 contradicts this.

Reply:
Not always. I'm referring to https://html.spec.whatwg.org/multipage/document-sequences.html#creating-a-new-browsing-context step 15.2 "Set document's [policy container](https://html.spec.whatwg.org/multipage/dom.html#concept-document-policy-container) to a [clone](https://html.spec.whatwg.org/multipage/browsers.html#clone-a-policy-container) of creator's [policy container](https://html.spec.whatwg.org/multipage/dom.html#concept-document-policy-container).".

Should this say "can inherit" instead?

Reply:
I'm going to change this to "can inherit" and resolve this conversation. Feel free to re-open if you think there's better wording for this.